# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../cyrus-sasl2/gcc-4.0.patch
# Copyright (C) 2004 - 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- cyrus-sasl-2.1.20/lib/client.c.vanilla	2005-09-02 15:51:21.000000000 +0000
+++ cyrus-sasl-2.1.20/lib/client.c	2005-09-02 15:51:56.000000000 +0000
@@ -61,7 +61,7 @@
 
 static cmech_list_t *cmechlist; /* global var which holds the list */
 
-static sasl_global_callbacks_t global_callbacks;
+sasl_global_callbacks_t global_callbacks;
 
 static int _sasl_client_active = 0;
 
